dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -552,13 +557,39 @@ export default defineComponent({
         this.postEuTaxonomyDataForFinancialsProcessed = false;
         this.messageCount++;
 
+        // JSON.parse/stringify used to clone the formInputsModel in order to stop Proxy refreneces
```

Comment:
I find you should put this added model conversion code into a function for the sake of readability